app/assets/javascripts/discourse/app/templates/components/user-status-picker.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="user-status-picker-wrap">
+  <div class="emoji-picker-anchor user-status-picker {{if this.isFocused "focused"}}">
+    <button
+      type="button"
+      class="btn-emoji btn-flat"
+      onclick={{action "toggleEmojiPicker"}}
+      {{on "focus" this.focus}}
+      {{on "blur" this.blur}}
+    >
+      {{#if @emoji}}
+        {{html-safe this.emojiHtml}}
+      {{else}}
+        {{d-icon "discourse-emojis"}}
```

Comment:
I think we should use the default icon right away here, instead of `discourse-emojis`, especially if using an emoji is not optional.

Reply:
I deliberately build in this behavior, put some time and efforts into it. May we merge it in and try it for a while first? We can always get rid of it in case we don't like it.